src/client.ts

```diff
@@ -4212,6 +4212,18 @@ export class MatrixClient extends EventEmitter {
             info: info,
             body: text,
         };
+
+        const thread = this.getRoom(roomId)?.threads.get(threadId);
+        if (thread) {
+            content["m.relates_to"] = {
+                "rel_type": RelationType.Thread,
+                "event_id": threadId,
+                "m.in_reply_to": {
+                    "event_id": thread.replyToEvent.getId(),
+                },
+            };
```

Comment:
kind of surprised there isn't a helper to generate this given a `thread`

Reply:
This is done in the composer for all other event types. But stickers work slightly differently and it constructs the event in the "send" method.
If that starts being used elsewhere in the SDK I will create a helper to avoid duplication